ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -892,9 +893,26 @@ def serialize_module(
         if additional_files:
             exec_config["module_data"] = {"additional_files": cast(JsonObject, additional_files)}
 
+        dataset = None
+        try:
+            sandbox_module_path = f"{module}.sandbox"
+            sandbox_module = importlib.import_module(sandbox_module_path)
+            if hasattr(sandbox_module, "dataset"):
+                dataset_attr = getattr(sandbox_module, "dataset")
+                if dataset_attr:
+                    dataset = []
+                    for inputs_obj in dataset_attr:
+                        if hasattr(inputs_obj, "__dict__"):
```

Comment:
make this safer by checking `isinstance(inputs_obj, BaseInputs)`. Then, we should use `json.loads(json.dumps(inputs_obj, cls=DefaultStateEncoder))` to get the serialized dict